graphs/rdf-clear-insert-jsonld.fbp

```diff
@@ -0,0 +1,34 @@
+# rdf-clear-insert-jsonld.fbp
+
+# Takes parsed JSON-LD and clears and inserts it into the target-graph (IRI) at the sparql-endpoint.
+
+# Exports
+INPORT=request.URL:SPARQL_ENDPOINT
```

Comment:
The spec uses the term endpoint as in "SPARQL endpoint...The URI at which a SPARQL Protocol service listens for requests from SPARQL Protocol clients.", why do you suggest using the term server-uri?

https://www.w3.org/TR/2013/REC-sparql11-protocol-20130321/#terminology